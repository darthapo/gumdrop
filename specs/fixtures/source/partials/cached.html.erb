  <%= render 'params', name:'Matt', :cache=>true %>
  <%= render 'params', name:'Dan' %>
  <%= render 'params', name:'Sam' %>
  <%= render 'params', name:'Other', :cache=>false %>
