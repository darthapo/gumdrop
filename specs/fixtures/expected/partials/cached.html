<body class="site">  <div>Matt</div>
  <div>Matt</div>
  <div>Matt</div>
  <div>Other</div>
</body>